<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestForm</class>
 <widget class="QWidget" name="TestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{background-color: transparent; border: 0px solid black;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_ok">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
                            color:#ffffff;
                            background-color:#424956;
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
                            QPushButton:hover{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>正常</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_ng7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
                            color:#ffffff;
                            background-color:#424956;
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
                            QPushButton:hover{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>NG7</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_ng6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
                            color:#ffffff;
                            background-color:#424956;
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
                            QPushButton:hover{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>NG6</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_ng8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
                            color:#ffffff;
                            background-color:#424956;
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
                            QPushButton:hover{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>NG8</string>
       </property>
       <property name="shortcut">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_ng2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
                            color:#000000;
	background-color: rgb(229, 229, 229);
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
                            QPushButton:hover{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>NG2</string>
       </property>
       <property name="shortcut">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_ng5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
                            color:#ffffff;
                            background-color:#424956;
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
                            QPushButton:hover{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>NG5</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_ng1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
                            color:#ffffff;
                            background-color:#424956;
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
                            QPushButton:hover{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>登入框</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_ng3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
                            color:#ffffff;
                            background-color:#424956;
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
                            QPushButton:hover{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>NG3</string>
       </property>
       <property name="shortcut">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_ng4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>80</horstretch>
         <verstretch>80</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { 
                            color:#ffffff;
                            background-color:#424956;
                            border-style: outset;
                            border-width:2px;
                            border-radius:10px;
                            border-color: beige;
                            font: bold 16px;
                            padding:6px;
                            }
                            QPushButton:hover{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:pressed{
                            color:#ffffff;
                                background-color:#ff5500
                            }
                            QPushButton:!enabled{
									
								color: rgb(235, 235, 235);
								
	background-color: rgb(143, 143, 143);
							}</string>
       </property>
       <property name="text">
        <string>NG4</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
